consultation_analyser/support_console/jinja2/support_console/consultations/index.html

```diff
@@ -36,6 +36,11 @@ <h1 class="govuk-heading-l">{{ page_title }}</h1>
         'text': "Generate dummy consultation",
         'name': "generate_dummy_consultation"
       }) }}
+      {{ govukButton({
```

Comment:
should this only appear if env=local?

Reply:
Am keen to run on dev, so I think fine for not production.